com.unity.netcode.gameobjects/Tests/Runtime/NetworkTransform/NetworkTransformPacketLossTests.cs

```diff
@@ -5,6 +5,9 @@
 using UnityEngine;
 using UnityEngine.TestTools;
 
+// TODO: Rewrite test to use the tools package. Debug simulator not available in UTP 2.X.
+#if !UTP_TRANSPORT_2_0_ABOVE
```

Comment:
I can include this update in #2777

Reply:
So sorry Simon... 
Just realized this requires us to include the tools package in our asmdef files which most likely should be a separate PR.
I will just include your #if check my PR, but we should handle the update in a separate PR.